zipline/finance/controls.py

```diff
@@ -425,3 +430,40 @@ def validate(self,
         """
         if _account.leverage > self.max_leverage:
             self.fail()
+
+
+class MinLeverage(AccountControl):
+    """
+    AccountControl representing a limit on the minimum leverage allowed
+    by the algorithm after a threshold period of time.
+    """
+
+    @expect_types(
+        __funcname='MinLeverage',
+        min_leverage=(int, float),
+        deadline=pd.datetime
+    )
+    @expect_bounded(__funcname='MinLeverage', min_leverage=(0, None))
+    def __init__(self, min_leverage, deadline):
+        """
```

Comment:
We generally don't put docstrings on `__init__` functions; we use the class docstring documenting the parameters to a class and describing what it's for.